packages/manager/modules/netapp/src/dashboard/volumes/create/controller.js

```diff
@@ -101,7 +101,7 @@ export default class VolumeCreateCtrl {
   }
 
   goBack() {
-    this.trackClick('create::cancel');
+    this.trackClick('button::add_volume::cancel');
```

Comment:
Keep CREATE_VOLUME_TRACKING_PREFIX inside constant
```suggestion
     this.trackClick(`${CREATE_VOLUME_TRACKING_PREFIX}::cancel`);
```